app.js

```diff
@@ -13,7 +13,9 @@ const path         = require('path');
 mongoose
   .connect('mongodb://localhost/starter-code', {useNewUrlParser: true})
   .then(x => {
-    console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`)
+    console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`);
+    
+    const seeds = require('./bin/seeds.js');
```

Comment:
Not really sure if this was the best way to call and seed the database.

Reply:
You can make a script in your package.json to run just that file when you need to. You don't have to require it in app.js. Just connect to the databsase in the seed file also!
Good job!